### DISCUSSION TITLE
【求助】急急急！！绳匠们，帮帮我！

### DISCUSSION THREAD

**Discussion by Rolmans** [share121/inter-knot#885] [其他]
<img width="1280" alt="屏幕截图 2024-08-10 180117" src="https://github.com/user-attachments/assets/b9eea6ff-b851-436b-982f-3b20b3e5267e">

本褪色者在交界地的湖之利耶尼亚迷路，听说各位绳匠上知天文，下至地理，无所不知。所以怎么才能到地图上那座岛！！急！

**Comment by March-777th** [comment]
小三月不知道呦
![三月挠头](https://github.com/user-attachments/assets/e7164784-a9cc-4901-ae64-5cea60ea9a7d)

**Comment by Ikaros423** [comment]
来了，这岛要走四钟楼的传送门

**Comment by WLMRMZ** [comment]
这个委托，我不会（叉腰）（自豪）

**Comment by SAD-BYTE** [comment]
我
